sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/api/exception/ValidationExceptionMapper.java

```diff
@@ -0,0 +1,19 @@
+package io.stargate.sgv2.docsapi.api.exception;
+
+import io.stargate.sgv2.docsapi.api.common.exception.model.dto.ApiError;
+import javax.validation.ValidationException;
+import javax.ws.rs.core.Response;
+import org.jboss.resteasy.reactive.RestResponse;
+import org.jboss.resteasy.reactive.RestResponse.ResponseBuilder;
+import org.jboss.resteasy.reactive.server.ServerExceptionMapper;
+
+/** Simple exception mapper for the {@link ValidationExceptionMapper}. */
+public class ValidationExceptionMapper {
+
+  @ServerExceptionMapper
+  public RestResponse<ApiError> validationException(ValidationException exception) {
```

Comment:
are you sure this is thrown on the invalid input? I am afraid we would cover way bigger scope with this one.. Thinking that the `ResteasyReactiveViolationException` could be the one we should target. Not sure?

Reply:
I changed to ResteasyReactiveViolationException - for some reason I wasn't able to find it anywhere in my search through quarkus code before, thanks for finding it!

Reply:
Maybe @Tatu is right and this was peeking into internal api.. As I wrote in the comment, was not sure myself as well.. Let's stick to the `javax.validation.ConstraintViolationException` which is a parent class of this one.. Not sure, maybe also the first approach was OK..